vertical-pod-autoscaler/recommender/util/histogram.go

```diff
@@ -157,25 +161,40 @@ func (h *histogram) IsEmpty() bool {
 }
 
 func (h *histogram) String() string {
-	lines := []string{"%-tile\tvalue"}
+	lines := []string{
+		fmt.Sprintf("minBucket: %d, maxBucket: %d, totalWeight: %.3f",
+			h.minBucket, h.maxBucket, h.totalWeight),
+		"%-tile\tvalue",
+	}
 	for i := 0; i <= 100; i += 5 {
 		lines = append(lines, fmt.Sprintf("%d\t%.3f", i, h.Percentile(0.01*float64(i))))
 	}
 	return strings.Join(lines, "\n")
 }
 
-// HistogramsEqual is a helper function for comparing 2 histograms.
-func HistogramsEqual(histogram1 Histogram, histogram2 Histogram) bool {
-	h1 := (histogram1).(*histogram)
-	h2 := (histogram2).(*histogram)
-	if h1.options != h2.options || h1.minBucket != h2.minBucket || h1.maxBucket != h2.maxBucket {
+func (h *histogram) Equals(other Histogram) bool {
+	h2, typesMatch := (other).(*histogram)
```

Comment:
The object for type assertion is not typically enclosed in paranthesis: https://tour.golang.org/methods/15.

Reply:
Done.